packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+
+import credentialBanner from './componenets/credential-banner';
+
+import component from './component';
+
+const moduleName = 'ovhManagerPciStoragesContainersAddCreateLinkedUser';
+
+angular
+  .module(moduleName, ['pascalprecht.translate', 'oui', credentialBanner])
+  .component('createLinkedUser', component)
```

Comment:
**issue(blocking)**: Component must be prefixed (prevent using too generic name for component).

Reply:
right thanks